HippUnfoldHCP/HippUnfoldHCP.sh

```diff
@@ -0,0 +1,78 @@
+    exit 1
+fi
+
+if [ ! -f "$T2wImage" ]; then
+    echo "Error: T2w image not found at $T2wImage" >&2
+    exit 1
+fi
+
+
+mkdir -p "$HippUnfoldT1wFolder" "$HippUnfoldT2wFolder" "$HippUnfoldT1wT2wFolder"
+
+ln -s "$T1wImage" "$HippUnfoldT1wFolder/s_${Subject}_T1w_acpc_dc_restore.nii.gz"
+ln -s "$T2wImage" "$HippUnfoldT1wFolder/s_${Subject}_T2w_acpc_dc_restore.nii.gz"
+ln -s "$T1wImage" "$HippUnfoldT2wFolder/s_${Subject}_T1w_acpc_dc_restore.nii.gz"
+ln -s "$T2wImage" "$HippUnfoldT2wFolder/s_${Subject}_T2w_acpc_dc_restore.nii.gz"
+ln -s "$T1wImage" "$HippUnfoldT1wT2wFolder/s_${Subject}_T1w_acpc_dc_restore.nii.gz"
+ln -s "$T2wImage" "$HippUnfoldT1wT2wFolder/s_${Subject}_T2w_acpc_dc_restore.nii.gz"
+
+log_Msg "Created folder structure under $HippUnfoldDIR and copied T1w and T2w images"
+log_Msg "Starting HippUnfold pipeline for subject: $Subject"
+  
+
+log_Msg "Running T1w HippUnfold for subject: $Subject"                                                                                                      #Seriously: don't put a $ here and don't capitalize S in subject...    #Seriously: don't put a $ here and don't capitalize S in subject...
+apptainer run --bind $StudyFolder -e $HIPPUNFOLDPATH $HippUnfoldT1wFolder $HippUnfoldT1wFolder participant --modality T1w --path-T1w $HippUnfoldT1wFolder/s_{subject}_T1w_acpc_dc_restore.nii.gz --path-T2w $HippUnfoldT1wFolder/s_{subject}_T2w_acpc_dc_restore.nii.gz --cores all --force-output --generate_myelin_map --output-density native 512 2k 8k 18k
+log_Msg "T1w HippUnfold completed."
+log_Msg "Running T2w HippUnfold for subject: $Subject"                                                                                                      #Seriously: don't put a $ here and don't capitalize S in subject...    #Seriously: don't put a $ here and don't capitalize S in subject...
+apptainer run --bind $StudyFolder -e $HIPPUNFOLDPATH $HippUnfoldT2wFolder $HippUnfoldT2wFolder participant --modality T2w --path-T1w $HippUnfoldT2wFolder/s_{subject}_T1w_acpc_dc_restore.nii.gz --path-T2w $HippUnfoldT2wFolder/s_{subject}_T2w_acpc_dc_restore.nii.gz --cores all --force-output --generate_myelin_map --output-density native 512 2k 8k 18k
+log_Msg "T2w HippUnfold completed."
+log_Msg "Running T1wT2w HippUnfold for subject: $Subject"                                                                                                            #Seriously: don't put a $ here and don't capitalize S in subject...       #Seriously: don't put a $ here and don't capitalize S in subject...
+apptainer run --bind $StudyFolder -e $HIPPUNFOLDPATH $HippUnfoldT1wT2wFolder $HippUnfoldT1wT2wFolder participant --modality T2w --path-T1w $HippUnfoldT1wT2wFolder/s_{subject}_T1w_acpc_dc_restore.nii.gz --path-T2w $HippUnfoldT1wT2wFolder/s_{subject}_T2w_acpc_dc_restore.nii.gz --cores all --force-output --generate_myelin_map --output-density native 512 2k 8k 18k --force-nnunet-model T1T2w
+log_Msg "T1wT2w HippUnfold completed."
```

Comment:
Is someone going to edit this to have the possibility of running with `--use-conda` instead of their container?

The warnings probably deserve a real newline instead spaces after the `log_Msg`, and we may not need quite that many repetitions.